front/src/containers/StudyPage/components/StudyPageHeader.tsx

```diff
@@ -235,13 +235,9 @@ class StudyPageHeader extends React.Component<StudyPageHeaderProps, StudyPageHea
 
             }
         })
-
-
-
-        this.props.studyRefetch();
+        .then(()=>this.props.studyRefetch())
+        .then(()=>refetch())
```

Comment:
Thanks for doing this fix. I think this will be much more reliable.  Any time you have a fixed timeout some network thing will happen and it'll end up flaky